cmd/bnsd/app/app_performance_test.go

```diff
@@ -80,7 +81,10 @@ func BenchmarkBNSD(b *testing.B) {
 					},
 				}
 
-				const nonce = 0 // ?!?!?!?
+				nonce, err := getNonce(wapp, alice.PublicKey().Condition().Address())
```

Comment:
Got this compile error:
```
cmd/bnsd/app/app_performance_test.go:84:27: cannot use wapp (type weavetest.WeaveApp) as type weave.ReadOnlyKVStore in argument to getNonce:
	weavetest.WeaveApp does not implement weave.ReadOnlyKVStore (missing Get method)
```

Reply:
Sorry, I forgot to push last changes Friday night.
Pull and it should run.

I also would like to use the increment integer, but using the query for the first nonce value should be negligable overhead. We need something like this for all queries as well (it just lets us re-use existing parsing logic rather than re-write it each time)